### ISSUE TITLE
[3.2.4-rc4] 2D pixel snap doesn't seem to work (reason: it was renamed)

### ISSUE THREAD

**Issue by snoopdouglas** [godotengine/godot#46822]
### Solved

TLDR: under the project settings, in *Rendering > 2D > Sampling*, enable 'Use Gpu Pixel Snap'.

Alternatively, in `project.godot` under `[rendering]`, add:

```
2d/snapping/use_gpu_pixel_snap=true
```

...and remove:

```
quality/2d/use_pixel_snap=false
```

---

**Godot version:**
3.2.4.rc4.official

**OS/device including version:**
Ubuntu 20.04, GLES2, GeForce 1660 super

**Issue description:**
In rc4, I noticed one of my sprites was missing its bottom row of pixels. The sprite is normally viewed through a sub-viewport, and I found this could be fixed by incrementing `position.y` by 0.5.

This doesn't happen when the window's stretch mode is set to 'viewport' - only when I've set up a viewport and a sprite with a `ViewportTexture`.

#### rc4

![Screenshot from 2021-03-09 14-15-23](https://user-images.githubusercontent.com/4892574/110483738-ec933780-80e1-11eb-80b5-b683857a028c.png)

#### rc3

![Screenshot from 2021-03-09 14-15-10](https://user-images.githubusercontent.com/4892574/110483765-f1f08200-80e1-11eb-8f81-dd6af053d164.png)

**Steps to reproduce:**
* Add a viewport with a sprite as a child
* Set the viewport to use v-flip
* Add an outer sprite to display the viewport
* Adjust the viewport's sprite's position and note the change

**Minimal reproduction project:**
[rc4-pixel-snap.zip](https://github.com/godotengine/godot/files/6109109/rc4-pixel-snap.zip)

**Comment by akien-mga** [comment]
The pixel snap option was renamed in 3.2.4 RC 4 so you need to re-enable it:

> Note that the project settings from the `rendering/quality/2d` section have now been moved to `rendering/2d`, so if you used any of those, you will need to re-enable them under the new section in 3.2.4.

**Comment by snoopdouglas** [comment]
That seems to have fixed it, thanks!

On a side note, the inclusion of *all* of the changes since 3.2.3 on the RC announcement makes things like this tricky to pick out. Could I suggest that changes specifically since the previous RC get their own section?

**Comment by akien-mga** [comment]
There's a link to the changelog since the previous build: https://github.com/godotengine/godot/compare/4f891b706027dc800f6949bec413f448defdd20d...dc99f04d51d6556e5ba4d9cfcce8117d168ac6f1

But yeah I agree it's easy to miss relevant information. There's just so much information already that making a dedicated session for just what changed since the previous build would make the post quite lengthy, and would require a lot of work for every build. Currently I skim the list of changes for some of the highest level changes to add to the list, but a dedicated section would have to go more into the details and practically rehash half of the detailed changelog.

I do try to highlight the main user facing changes that should be taken into account but again, the post is already so loaded with info (and there's so much important stuff that I couldn't feature) that it's very easy to miss.

**Comment by lawnjelly** [comment]
It may even be worth leaving this open for a few days so that others will see the issue if they have the same problem.

**Comment by snoopdouglas** [comment]
> a dedicated section would have to go more into the details and practically rehash half of the detailed changelog

That's fair enough - with the patch rate as high as it is, I'd imagine just putting together the announcement is hard enough.

An alias system sounds like a brilliant idea :+1:

**Comment by akien-mga** [comment]
Yeah let's keep it open for now as users often search for duplicates but not among closed issues.

**Comment by akien-mga** [comment]
Fixed by #46827.
